payments-core/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -631,9 +631,9 @@ internal class StripePaymentController internal constructor(
         internal const val SOURCE_REQUEST_CODE = 50002
 
         /**
-         * List of payment methods not to override the return_url to [DefaultReturnUrl]
+         * List of payment methods to override the return_url to [DefaultReturnUrl]
          */
-        internal val RETURN_URL_OVERRIDE_BLOCKLIST = setOf(PaymentMethod.Type.WeChatPay)
+        internal val RETURN_URL_OVERRIDE_ALLOWLIST = setOf(PaymentMethod.Type.Card)
```

Comment:
I think there are other payment methods that will need to be in this set, I just don't know which. 😬

Reply:
reverted to blockList after offline discussion, will check with payment-flows team